src/Microsoft.Health.Fhir.Core/Features/Persistence/IFhirDataStore.cs

```diff
@@ -22,8 +22,6 @@ public interface IFhirDataStore
 
         Task HardDeleteAsync(ResourceKey key, CancellationToken cancellationToken);
 
-        Task UpdateSearchParameterHashBatchAsync(IReadOnlyCollection<ResourceWrapper> resources, CancellationToken cancellationToken);
-
```

Comment:
@feordin I removed this, since it has no usages and it is easier to update the search parameter hash at the same time as the indices - is there anything I'm missing/any reason to keep it?

Reply:
We will only end up using this is if we implement the performance enhancement which compares preivous indices to the newly extracted indices, and then determine that the only update required is the hash value.  We can add this method back in if we ever implement that.  So it is fine to remove it.

Reply:
Ah, I see - thank you! I'll make a note about it in [AB#79648](https://microsofthealth.visualstudio.com/Health/_workitems/edit/79648).